src/components/TodoScreen.js

```diff
@@ -0,0 +1,209 @@
+import React, { useState, useEffect, useRef } from "react";
+import styled from "styled-components";
+
+import StatusBar from "./TodoStatusBar";
+import Title from "./TodoTitle";
+import ScrollView from "./TodoScrollView";
+
+import * as Icons from './Icons';
+
+const ScreenWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  align-items: center;
+  width: 300px;
+  height: 650px;
+  overflow: hidden;
+
+  &::-webkit-scrollbar {
+    display: none;
+  }
+
+  & ::-moz-scrollbar {
+    display: none;
+  }
+
+  & {
+    overflow: auto;
+    -ms-overflow-style: none;
+    scrollbar-color: transparent transparent;
+  }
+
+  color: black;
+  text-align: center;
+  border-radius: 30px;
+  background: #abc2d0;
+  box-shadow: 0 1rem 4rem hsl(0 0% 0% / 60%);
+
+  ${({ darkMode }) =>
+    darkMode &&
+    `
+    background: #262628;
+  `}
+`;
+
+const ScreenHeader = styled.header`
+  flex: 1;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  width: 100%;
+`;
+
+const ScreenContent = styled.main`
+  flex: 7;
+  width: 100%;
+  z-index: 1000 !important;
+`;
+
+const ScreenFooter = styled.footer`
+  flex: 0.9;
+  width: 100%;
+  background: white;
+
+${({ darkMode }) =>
+  darkMode &&
+  `
+  background: #2c2c2e;
+`}
+`;
+
+const InputWrapper = styled.main`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  margin: 8px 13px 0px 13px;
+`;
+
+const TodoInput = styled.input`
+  text-align: left;
+  border-color: transparent;
+  border-radius: 10px;
+  outline: none;
+  width: 220px;
+  height: 30px;
+  padding: 0 0 0 10px;
+  background-color: #ededed;
+  font-size: 11px;
+  font-weight: 400;
+
+  ${({ darkMode }) =>
+    darkMode &&
+    `
+    color: white;
+    background: #38383a;
+  `}
+`;
+
+const TodoInputForm = styled.form`
+  z-index: 1000 !important;
+`;
+
+const TodoInputSend = styled.div`
+  width: 26px;
+  height: 26px;
+  object-fit: contain;
+  opacity: 0.5;
+  z-index: 1000 !important;
+`;
+
+const TodoInputSendImg = styled.img`
+  width: 26px;
+  height: 26px;
+  object-fit: cover;
+`;
+
+const initialTodos = [];
+
+const TodoScreen = ({ darkMode, ...props }) => {
+  const [todos, setTodos] = useState(
+    () =>
+      JSON.parse(window.localStorage.getItem("localStorageTodos")) ||
+      initialTodos
+  );
```

Comment:
arrow function이 꼭 필요할까요?
```suggestion
  const [todos, setTodos] = useState(
      JSON.parse(window.localStorage.getItem("localStorageTodos")) ||
      initialTodos
  );
```

Reply:
디버깅 끝나고 삭제했다고 생각했는데 남아있었네요....
부끄럽습니다ㅠ

Reply:
미처 생각하지 못하고 지나친 실수네요ㅠ 꼭 신경쓰겠습니다.
별개로 React로 localStorage save-load를 구현해 보니
어떤 정보를 store해야 할 지 명확하게 받아들이고 구현할 수 있어 편했던 것 같아요! (TMI)